### DISCUSSION TITLE
A real compiler and a real language matrix

### DISCUSSION THREAD

**Discussion by ghost** [community/community#65803] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Show & Tell

### Body

Hi everybody i am new here i would like to share what I have been thinking about for years please let me know if your interested in my projects thanks!

I have developed a sigil matrix using sequential numbering but other systems work as well. 

```
         ! @ # $ % ^ & * ( !)
!! !@ !# !$ !% !^ !& !* !( @)
@! @@ @# @$ @% @^ @& @* @( #)
#! #@ ## #$ #% #^ #& #* #( $)
$! $@ $# $$ $% $^ $& $* $( %)
%! %@ %# %$ %% %^ %& %* %( ^)
^! ^@ ^# ^$ ^% ^^ ^& ^* ^( &) 
&! &@ &# &$ &% &^ && &* &( *) 
*! *@ *# *$ *% *^ *& ** *( ()
(! (@ (# ($ (% (^ (& (* (( !))
```

**Comment by ghost** [comment]
thank you very much.

**Comment by Akash1134** [comment]
Hi @ApexExpress , we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
